server/models/groups/createGroup.js

```diff
@@ -0,0 +1,53 @@
+const { pool } = require('../database')
+
+/**
+ *z
+ * @param {*} param0
+ * @param {String} param0.username
+ * @param {Object} param0.body
+ * @return {Promise}
+ *
+ */
+
+function createGroup({ username, body }) {
+  return new Promise((resolve, reject) => {
+    const { group_name: groupName, confidential, members } = body
+    pool.query(
+      `INSERT INTO cl_judge.groups (group_name,creator,confidential,member_count)
```

Comment:
It should be `INSERT INTO groups` , following the way we have used in all the queries earlier. See [model for signup API](https://github.com/Cyber-Labs/cl-judge/blob/master/server/models/auth/signup.js) for reference